plugin.json

```diff
@@ -0,0 +1,13 @@
+{
+    "id": "generalized-linear-models",
+    "version": "0.0.1",
+    "meta": {
+        "label": "Generalized Linear Models",
+        "description": "Generalization of linear models that allows for response variables with error distributions that are not normally distributed",
+        "author": "Dataiku (Matthew Galloway)",
+        "icon": "icon-briefcase",
+        "tags": ["Machine Learning"],
+        "url": "",
```

Comment:
For this release (not on the store, just v1 testing) we are going to avoid producing official documentation until we have gone through the alpha test phase.

Reply:
That's OK not to have official documentation for the alpha. Having said that, you can perfectly set the documentation URL in advance, in preparation for when it is ready.

Reply:
In the end of the day, this documentation page will need to be filled before the plugin can be published.

Reply:
5d029ac3f9b474f2c185206354f83027eb87a089

Reply:
SupportLevel can be one of https://github.com/dataiku/dip/blob/55ba6aa11fd1c28f30fe4500ac7836c82b635bb5/src/main/platypus/static/dataiku/js/admin/plugins.js#L1054

Experimental is not part of that list. In your case, my understanding is that it is NOT_SUPPORTED

Reply:
[7697b0b](https://github.com/dataiku/dss-plugin-generalized-linear-models/commit/7697b0b01e6ce58324f8f30ea9bd70680c0cd6fa)